docs/newuser.md

```diff
@@ -1,29 +1,41 @@
 # Flow for new users
-This document will go over the flow for getting access to our various services
+This document will go over the flow for getting access to our various services if you are finding difficulties with the specifics of the tasks outlined in this document please see the more detailed [Developer Docs]()
 
-## Publicly Accessible
+## Office 365 Account
+[Office 365](https://portal.office.com), these are provided by emailing DigitalCapabilitiesPMO@homeoffice.gsi.gov.uk, this is not managed by us. Please note that for Single Sign On (SSO) which is using your Office 365 account to log onto other services pop-ups will need to be enabled on your computer.
+
+## VPN
+The VPN is used to be able to get through to our multiple AWS environments as well as other data centers. The profiles can be downloaded from [authd](https://authd.digital.homeoffice.gov.uk) which you can access using your office 365 SSO. The following services can be only accessed when connected to the Dev VPN profile.
+
```

Comment:
I'm thinking we should also note that access to London stuff (acp drone, platform hub etc.) requires the London VPNs through here `https://access-acp.digital.homeoffice.gov.uk` 
What do you think?

Reply:
Given these docs will be solely for ACP London (I assume?) we should just have the London VPN link here.